spec/models/company_branch_spec.rb

```diff
@@ -0,0 +1,5 @@
+require 'rails_helper'
+
+RSpec.describe CompanyBranch, type: :model do
+  pending "add some examples to (or delete) #{__FILE__}"
```

Comment:
Please add some specs in all the models, don't leave them pending

Reply:
It is pending because we still do not have the logic of the models, but later the tests will be done.